bigchaindb/commands/utils.py

```diff
@@ -20,6 +20,10 @@ def start_rethinkdb():
 
     for line in proc.stdout:
         if line.startswith('Server ready'):
+            # FIXME: seems like tables are not ready when the server is ready,
```

Comment:
https://rethinkdb.com/api/python/wait/

`database.wait` with `all_replicas_ready` should be enough but you can also run the wait on the tables just to be sure